docs/sources/tempo/setup/operator/multitenancy.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Setup multi-tenancy
+description: Set up multi-tenancy
+menuTitle: Multi-tenancy
+weight: 250
+aliases:
+- /docs/tempo/operator/multitenancy
+---
+
+# Multi-tenancy
+
+Tempo is a multi-tenant distributed tracing backend. It supports multi-tenancy through the use of a header: `X-Scope-OrgID`.
+See [multi-tenancy docs]({{< relref "../../operations/multitenancy" >}}) for more details. This document outlines how to deploy and use multi-tenant Tempo with the Operator.
+
+## Multitenancy without authentication
+
+The following example CR deploys multi-tenant Tempo instance. Note that Jaeger query is not tenant aware and therefore is not supported in this configuration.
+
+```yaml
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+spec:
+  tenants: {}
+  storage:
+    secret:
+      name: minio-test
+      type: s3
+  storageSize: 1Gi
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+```
+
+## OIDC authentication with static RBAC
+
+On Kubernetes the multi-tenant Tempo instance uses OIDC authentication and static RABC authorization defined in the CR.
+The instance should be accessed through service `tempo-simplest-gateway` which handles authentication and authorization.
+The service exposes Jaeger query API and OpenTelemetry gRPC (OTLP) for trace ingestion.
+The Jaeger UI can be accessed at `http://<exposed gateway service>:8080/api/traces/v1/<tenant-name>/search`
+
+```yaml
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+spec:
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+    gateway:
+      enabled: true
+  storage:
+    secret:
+      type: s3
+      name: minio-test
+  storageSize: 200M
+  tenants:
+    mode: static
+    authentication:
+      - tenantName: test-oidc
+        tenantId: test-oidc
+        oidc:
+          issuerURL: http://dex.default.svc.cluster.local:30556/dex
+          redirectURL: http://tempo-simplest-gateway.default.svc.cluster.local:8080/oidc/test-oidc/callback
+          usernameClaim: email
+          secret:
+            name: oidc-test
+    authorization:
+      roleBindings:
+      - name: "test"
+        roles:
+        - read-write
+        subjects:
+        - kind: user
+          name: "admin@example.com"
+      roles:
+      - name: read-write
+        permissions:
+        - read
+        - write
+        resources:
+        - traces
+        tenants:
+        - test-oidc
+```
+
+* The secret `oidc-test` defines fields `clientID`, `clientSecret` and `issuerCAPath`.
+* The RABC gives tenant `test-oidc` read and write access for traces. 
+
+## OpenShift
+
+On OpenShift the authentication and authorization does not require any 3rd party service dependencies.
+The authentication uses OpenShift OAuth (the user is redirected to the OpenShift login page) and 
+authorization is handled through `SubjectAccessReview` (SAR).
```

Comment:
nit: 
[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.
```suggestion
The authentication uses OpenShift OAuth (the user is redirected to the OpenShift login page) and authorization is handled through `SubjectAccessReview` (SAR).
```